src/ui/MobileNavbar/Navbar.svelte

```diff
@@ -0,0 +1,84 @@
+<script lang="ts" context="module">
+  export const MOBILE_NAVBAR_LINKS = [
+    {
+      title: 'Market',
+      icon: 'market',
+      link: '/assets',
+    },
+    {
+      title: 'Chart',
+      icon: 'chart',
+      link: '/projects',
+      slug: '/bitcoin',
+    },
+    {
+      title: 'Watchlist',
+      icon: 'watchlist',
+      link: '/watchlists',
+    },
+    {
+      title: 'Insights',
+      icon: 'insights',
+      link: '/insights',
+    },
+    {
+      title: 'Menu',
+      icon: 'mobile-menu',
+      isMenuNav: true,
+    },
+  ]
+</script>
+
+<script lang="ts">
+  import Menu, { getFullLink } from './Menu.svelte'
+  import NavItem from './NavItem.svelte'
+
+  export let user: SAN.Author & { name?: string }
+  export let path = ''
+  export let isFullLink = false
```

Comment:
`export let link = MOBILE_NAVBAR_LINKS` since some links might be changed according to the platform they are used on (e.g. - on `Insights`  `'/insights'` will be changed to `'/'`)